Book-A-Desk.Api.Tests/TestServer.fs

```diff
@@ -0,0 +1,31 @@
+﻿module Book_A_Desk.Api.Tests.TestServer
+
+open System
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.Hosting
+open Microsoft.Extensions.DependencyInjection
+open Giraffe
+
+open Book_A_Desk.Api
+
+let private configureApp eventStore getOffices (app : IApplicationBuilder) =
+    let routes = Routes.provide eventStore getOffices
+    app.UseGiraffe routes.HttpHandlers
+
+let private configureServices (services : IServiceCollection) =
+    services.AddGiraffe() |> ignore
+    
+let createAndRun port eventStore getOffices =
```

Comment:
I'm not sure if this is the best way to test a ASP.NET and Giraffe, but it seems to work for me. I'd welcome any suggestions :)